tool/tsh/tsh_test.go

```diff
@@ -537,20 +537,30 @@ func approveAllAccessRequests(access services.DynamicAccess, stop <-chan struct{
 }
 
 // TestSSHAccessRequest tests that a user can automatically request access to a
-// ssh server using a search-based access request when "tsh ssh" fails with
+// ssh server using a resource access request when "tsh ssh" fails with
 // AccessDenied.
 func TestSSHAccessRequest(t *testing.T) {
```

Comment:
Can this be a `t.Parallel()` test? Looks like it takes 9-10s which is pretty long. 

Making it faster would be even better.

Reply:
I've made it parallel, unfortunately the logic is added in `onSSH` which is really hard to test without an auth, proxy, and node service running, the setup and teardown of those is what takes all the time. It will also run in parallel with all other package tests and isn't extremely cpu intensive so I don't think it will effect our overall test runtime too much.